lib/hid-sys/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "hid-sys"
-version = "0.0.1"
+version = "0.1.0"
```

Comment:
Please don't change the version number. I do that myself when I push things to crates.io.